debian/control

```diff
@@ -32,8 +32,8 @@ Depends: ${shlibs:Depends},
          python3,
          python3-samba,
          samba-dsdb-modules,
-         sssd,
-         sssd-dbus,
+         sssd | winbind,
+         sssd-dbus | winbind,
          apparmor,
          cifs-utils,
          krb5-user,
```

Comment:
I think we don’t need krb5-user for sssd use case. Mind adding the alternative there too?